data-generation/src/test/scala/SnapshotDeltaSpec.scala

```diff
@@ -16,6 +17,17 @@ class SnapshotDeltaSpec extends AnyFlatSpec with SparkTestWrapper {
     assert(graphs.graphs.length == 2)
   }
 
+  // Can be ignored if "it" is swapped with "ignore"
```

Comment:
```suggestion
```
Skjønte ikke denne kommentaren - `ignore should "also have the correct..."`?

Reply:
Vanligvis så er det jo 
```
it should "do stuff" { assert(doStuff) }
```

Testen feiler for øyeblikket så da feiler CI også. Da kan man bytte ut "it" med "ignore"
```
ignore should "do stuff" { assert(doStuff) }
```
Da vil testen ikke bli kjørt.

Reply:
Ah!!